build.gradle

```diff
@@ -42,9 +42,12 @@ dependencies {
 
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compile "com.corundumstudio.socketio:netty-socketio:1.7.7"
+
+    compile group: 'org.json', name: 'json', version: '20160212'
```

Comment:
I just need two functions from org.json; with Moshi you need a class which must have the same properties as the json, then use Moshi.Builder().build() etc...
It would complicate things. Besides the org.json jar it's only 50KB,